src/components/Markdown/detailsBlock.ts

```diff
@@ -0,0 +1,33 @@
+import { Marked } from '@ts-stack/markdown';
+
+export const DETAILS_BLOCK_RULE = /^\s*begin\[details\]\s+(.*?)\n([\s\S]*?)\n\s*end\[details\]\s*/;
```

Comment:
Right trim? We should do it in md file
```suggestion
export const DETAILS_BLOCK_RULE = /^\s*begin\[details\]\s+(.*?)\n([\s\S]*?)\n\s*end\[details\]/;
```